modules/product-alerts-magento-px/graphql/mutations/CustomerNotifyInStock.ts

```diff
@@ -0,0 +1,17 @@
+import field from '#ioc/graphql/field'
+import mutation from '#ioc/graphql/mutation'
+
+export default () =>
```

Comment:
Pu all the mutations into productAlerts subdirectory. Adn remove Customer form name. So this file would be in graphql/mutations/productAlerts/NotifyInStock.ts